pyproject.toml

```diff
@@ -1,22 +1,24 @@
 # The `dev-dependencies` section sets up tools for building `jj`
-# documentation. `poetry` will install these in a virtual environment.
-[tool.poetry]
-name = "jj-docs"
-package-mode = false
+# documentation. `uv` will install these in a virtual environment.
 
-[tool.poetry.dependencies]
-python = "^3.8"
+[project]
+name = "jj-docs"
+version = "0.1.0"
+requires-python = ">=3.8"
+dependencies = []
 
-[tool.poetry.group.dev.dependencies]
-# These can be updated with `poetry add`.
-mkdocs = ">=1.6,<1.7"
-mkdocs-material = ">=9.5.39,<9.6"
-# Versioning of documentation
-mike = ">=2.1.1,<3"
-# (Py)Markdown extensions
-mdx-truly-sane-lists = "^1.3"
-mdx-breakless-lists = "^1.0.1"
-# Allows setting up redirects when renaming docs files
-mkdocs-redirects = "^1.2.1"
-# Used for the CLI reference
-mkdocs-include-markdown-plugin = "^6.0.4"
+[tool.uv]
+dev-dependencies = [
+    # These can be updated with `uv add`.
+    "mkdocs<1.7,>=1.6",
+    "mkdocs-material<9.6,>=9.5.39",
```

Comment:
So, this goes a bit into superstitious territory, but could you run

    uv add --dev --upgrade mkdocs-material==9.5.39

I don't fully trust mkdocs-material updates, they sometimes break things (though I don't have any specific concerns about this version), and they change the website a lot, so I'd do them separately.

OTOH, I do trust `mike` updates to be quite careful about semver, so it might make sense to make the mike version `>=2.1.3`.

Reply:
Hmm, the command I gave does change the version specifier. It's probably fine to change it then, documenting my reluctance to update unless necessary.

Reply:
I completely agree re/ pinning mkdocs-material